### PR TITLE
[WIP][NOTEST][1LP] Sprout - UI for resolving stuck templates

### PR DESCRIPTION
**Pull request by mfalesni**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 330c8c043dd6004983f2913d2d1b2633468aa05f:

`sprout/appliances/models.py`:
- :bangbang: Line 616:41: **E701** *multiple statements on one line (colon)*
- :bangbang: Line 616:42: **E241** *multiple spaces after ':'*
- :bangbang: Line 616:42: **E272** *multiple spaces before keyword*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by gshefer** [comment]
@mfalesni Both WIP and RFR?

**Comment by mfalesni** [comment]
@gshefer Yes. I don't want PRT to test Sprout updates because it is pointless and wasting the cycles :) And NOTEST is not implemented in PRT so I have to combine it like this.

File: sprout/appliances/models.py [hunk 1]
```diff
@@ -607,6 +607,17 @@ def set_status(self, status):
             template.save()
             self.logger.info("{}: {}".format(self.pk, status))
 
+    @property
+    def possibly_bunk(self):
```
Comment (psav) [inline sprout/appliances/models.py]: **REQUIRED:** Can we have a docstring for this weirdly named method

Comment (mfalesni) [reply sprout/appliances/models.py]: Sure

**Pull request merged**